packages/react-native-win32/src/Libraries/Inspector/Inspector.win32.js

```diff
@@ -10,7 +10,7 @@
 
 'use strict';
 
-const Dimensions = require('../Utilities/Dimensions');
+// const Dimensions = require('../Utilities/Dimensions'); [Win32]
```

Comment:
intentional?

Reply:
We needed to patch out usages of a few things for rn-win32 due to not being supported. If we leave these here, eslint will warn about unused variables, so we comment it out and add brackets that have been helping us search for diffs quickly.

Reply:
thanks, makes sense!